modules/twinklexfd.js

```diff
@@ -807,9 +807,11 @@ Twinkle.xfd.callbacks = {
 		text += '}}';
 
 		if (params.delsortCats) { // Only for AFDs
+			text += '\n{{subst:Deletion sorting/multi';
 			params.delsortCats.forEach(function (cat) {
```

Comment:
this can be refactored to use `.join('|')` instead

Reply:
Done